website/source/docs/upgrading/plugins.html.md

```diff
@@ -35,4 +35,9 @@ with either the `plugin` or `mounts` parameter respectively.
 Until step 4, the mount will still use plugin_v1, and when the reload is
 triggered, Vault will kill plugin_v1’s process and start a plugin_v2 process.
 
-[plugin_reload_api]: /api/system/plugins-reload-backend.html
\ No newline at end of file
+-> **Important:** Plugin registration and reload of a new plugin binary must be 
```

Comment:
Just a reload is required on each node. Registration is done once on the active node in the primary datacenter

Reply:
Gotcha. I can update. Will this flow down to the secondaries? What about DR?

Reply:
Yes, replication will handle that.